### DISCUSSION TITLE
Contract?

### DISCUSSION THREAD

**Discussion by brandon-reinhart** [StarArawn/bevy_ecs_tilemap#201] [General]
StarArawn, would you be able to accept some amount of more direct funding or sponsorship to put more time into this project? (If that helps remove limiting factors on development.) I have no idea how to contact you.

**Comment by StarArawn** [comment]
Unfortunately I'm not sure this is a problem that can be solved in that way. I'm really busy with real life stuff at the moment between work and home. I'm available on the bevy discord as `StarToaster` if you'd like to talk more about it 1 on 1.
